workshops/migrations/0105_migrate_host_to_org.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2016-06-28 07:55
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workshops', '0104_auto_20160629_1247'),
+    ]
+
+    operations = [
+        migrations.RenameModel(
```

Comment:
The thing that startled me is that all previous migrations that use `apps.get_model('workshops', 'Host')` work just fine.